firebase-appdistribution/src/main/res/drawable/btn_disabled_stroked.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="rectangle">
+    <solid android:color="#FFFFFF" />
```

Comment:
Is it possible to make the background transparent? It looks a little weird since the parent container background isn't the same white, and the mocks show it as the same color as its parent.

Reply:
Done.